roles/mosquitto/defaults/main.yml

```diff
@@ -0,0 +1,4 @@
+mosquitto_install: True
+mosquitto_enabled: False
+mosquitto_user: admin
```

Comment:
Please change 'admin' to 'Admin' to be consistent with other Internet-in-a-Box playbooks, Thanks!